data_structures/linked-list/test_linked_list.py

```diff
@@ -43,14 +43,14 @@ def test_fail_search(small_ll):
     assert small_ll.find(6) is False
 
 
-# def test_append(small_ll):
-#     """Test append to end of list."""
-#     assert small_ll.append(5) == [1, 2, 3, 4, 5]
+def test_append(small_ll):
+    """Test append to end of list."""
+    assert small_ll.append(5) is not str
 
 
-# def test_before(small_ll):
-#     assert small_ll.insert_before(3, 10) == [1, 2, 10, 3, 4]
+def test_before(small_ll):
+    small_ll.insert_before(3, 10) == 3
```

Comment:
Same here. These methods do not return anything, so you should not be validating them in this way.